piper/buttondialog.py

```diff
@@ -153,6 +155,11 @@ def _init_other_buttons_ui(self, buttons):
                 self.listbox.select_row(row)
             i += 1
 
+        row = ButtonRow(_("Disable"), _("Misc mapping"), RatbagdButton.ActionType.NONE, 'disabled')
```

Comment:
I am not really happy with this, you are creating a new row for `RatbagdButton.ActionType.NONE` which only has and will ever have one action, disable.

Reply:
Me neither, I really don't like it. I tried to add an entry in the glade file similar to the keystroke entry, but that only results in errors. I'm not even entirely sure why, I suspect it this because of [this function](https://github.com/libratbag/piper/blob/master/piper/buttondialog.py#L132).
This function will failing trying to do [this](https://github.com/libratbag/piper/blob/master/piper/buttondialog.py#L194) because `_section` doesn't exist. Easiest workaround: add a Section.

Reply:
I don't think the code was designed with glade. It relies a lot on code.

Reply:
It uses a mix of both, which I suspect is the cause of my problem. [This](https://github.com/libratbag/piper/blob/master/data/ui/ButtonDialog.ui#L88) is the row for the keystroke.